.github/workflows/tests.yml

```diff
@@ -190,3 +190,83 @@ jobs:
           GITHUB_ORG_ID: ${{ secrets.DOTNET_SDK_GITHUB_ORG_ID }}
           GITHUB_REPO_ID: ${{ secrets.DOTNET_SDK_GITHUB_REPO_ID }}
           SSH_KEY: ${{ secrets.DOTNET_SDK_SSH_KEY }}
+
+  build-and-test-python-sdk:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Specify python version
+        uses: actions/setup-python@v4 
+        with:
+          python-version: '3.10' 
+
+      - name: Setup git
+        run: ./scripts/setup_git.sh
+        env:
+          GIT_USER_NAME: ${{ secrets.GIT_USER_NAME }}
+          GIT_USER_EMAIL: ${{ secrets.GIT_USER_EMAIL }}
+
+      # Commented out for now until we publish the SDK
+      - name: Clone the existing SDK
+        run: ./scripts/clone_sdk.sh
+        env:
+          GITHUB_ORG_ID: ${{ secrets.DOTNET_SDK_GITHUB_ORG_ID }}
```

Comment:
```suggestion
          GITHUB_ORG_ID: ${{ secrets.PYTHON_SDK_GITHUB_ORG_ID }}
```

Reply:
Same below

Reply:
Can you verify the  ORG_ID etc? 

Seems github action now fails with
(https://github.com/openfga/sdk-generator/runs/8006629984?check_suite_focus=true)
```
git@github***com: Permission denied (publickey)***
fatal: Could not read from remote repository***

Please make sure you have the correct access rights
and the repository exists***
Error: Process completed with exit code 128***
```

Reply:
Or we cannot run with python SDK until it is opened to public

Reply:
Done